### ISSUE TITLE
Version mismatches between package, CITATION file, and Zenodo

### ISSUE THREAD

**Issue by timothygebhard** [Lightning-AI/lightning#14559]
The current package version according to the GitHub releases (and PyPI) is 1.7.4, but the [CITATION.cff](https://github.com/Lightning-AI/lightning/blob/master/CITATION.cff) file still lists version 1.4, while pointing to a [Zenodo record](https://zenodo.org/record/3828935) that apparently hasn't been updated since version 0.7.6.

I can create a PR for the CITATION file (although maybe this could even be updated automatically by the CI?), but I guess Zenodo can only be fixed by a maintainer? 🙂

**Comment by Borda** [comment]
> I can create a PR for the CITATION file (although maybe this could even be updated automatically by the CI?)

Yes, this we shall keep in mind and update manually with every release, not sure if there is an automated way...

> but I guess Zenodo can only be fixed by a maintainer?

I am asking for this for more then a year :D
cc: @lexierule

**Comment by ziw-liu** [comment]
We are also looking into citing the package. At this point the CITATION file is very outdated.

**Comment by tchaton** [comment]
Hey @ziw-liu, could you make a PR to make it update to date ?

**Comment by Borda** [comment]
> Hey @ziw-liu, could you make a PR to make it update to date ?

The proper way would be enabled Zenodo
